docs/sw-update-conf.json

```diff
@@ -14,110 +14,81 @@
     },
     "restart": "environment"
   },
-  "octoprint": {
-    "pip": "https://github.com/mrbeam/OctoPrint/archive/{target_version}.zip",
-    "prerelease": false,
-    "develop":{
-      "prerelease": true
-    },
-    "beamos_date": {
-      "2021-06-11": {
+  "modules": {
+    "octoprint": {
+      "type": "github_release",
+      "develop": {
+        "type": "github_commit"
+      },
+      "beta": {
+        "prerelease_channel": "mrbeam2-{tier}",
+        "prerelease": true
+      },
+      "alpha": {
+        "prerelease_channel": "mrbeam2-{tier}",
         "prerelease": true
       }
-    }
-  },
-  "mrbeam": {
-    "name": " MrBeam Plugin",
-    "repo": "MrBeamPlugin",
-    "restart": "octoprint",
-    "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
-    "beamos_date": {
-      "2021-06-11": {
-        "branch": "mrbeam2-{tier}-buster",
-        "develop": {
-          "branch": "develop"
+    },
+    "mrbeam": {
+      "name": " MrBeam Plugin",
+      "repo": "MrBeamPlugin",
+      "restart": "octoprint",
+      "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
+      "dependencies": {
+        "mrbeam-ledstrips": {
+          "name": "MrBeam LED Strips",
+          "repo": "MrBeamLedStrips",
+          "pip": "https://github.com/mrbeam/MrBeamLedStrips/archive/{target_version}.zip",
+          "global_pip_command": true
+        },
+        "iobeam": {
+          "name": "iobeam",
+          "type": "bitbucket_commit",
+          "repo": "iobeam",
+          "pip": "git+ssh://git@bitbucket.org/mrbeam/iobeam.git@{target_version}",
+          "global_pip_command": true,
+          "api_user": "MrBeamDev",
+          "api_password": "v2T5pFkmdgDqbFBJAqrt"
+        },
+        "mrb_hw_info": {
+          "name": "mrb_hw_info",
+          "type": "bitbucket_commit",
+          "repo": "mrb_hw_info",
+          "pip": "git+ssh://git@bitbucket.org/mrbeam/mrb_hw_info.git@{target_version}",
+          "global_pip_command": true,
+          "package_name": "mrb-hw-info",
```

Comment:
Why is the "package_name" key added here and in Netconnectd Daemon but not for iobeam and mrbeam-ledstrips?

Reply:
because it is different to the repo name